jsonschema-hyperschema.xml

```diff
@@ -1330,29 +1330,38 @@ for varname in templateData:
             </section>
             <section title='"targetSchema" and HTTP' anchor="targetHTTP">
                 <t>
-                    The relationship between a resource's representation and HTTP requests and
-                    responses is determined by <xref target="RFC7231">RFC 7231, section 4.3.1 -
-                    "GET", section 4.3.4 "PUT", and section 3.1.4.2, "Content-Location"</xref>.
+                    "targetSchema" describes the resource on the other end of the link, while
```

Comment:
"other end" might be ambiguous here (the first think I actually thought of was "link" as in "satellite link" not "Web link"). I might say "target end" (or in semweb contexts, "object end").

Reply:
Yeah I kind of worried about that.  Thanks for the feedback, it helps and I'll update this.